.github/workflows/push.yml

```diff
@@ -1,41 +1,19 @@
-name: ci
+name: lint and format
 
-on:
-  push:
-    branches:
-      - 'main'
+on: push
 
 jobs:
-  docker:
+  lint_and_fmt:
     runs-on: ubuntu-20.04
-    permissions:
-      contents: read
-      packages: write
-
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+          node-version: 16
 
-      - name: Build and push
-        uses: docker/build-push-action@v3
-        with:
-          context: .
-          push: true
-          tags: ghcr.io/cutie-club/cutiebot:latest
-          platforms: linux/amd64, linux/arm64, linux/arm/v7
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+      - run: yarn install
+      - run: yarn lint
+      - run: yarn pretty-check
```

Comment:
is pretty-check neccesary due to `eslint-plugin-prettier` not that theres any harm having both, just curious

Reply:
No you're right, running both is extraneous really. I'll remove `pretty-check` from the job.